### PR TITLE
Add tests for changes to chocolatey module

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Adds tests for https://github.com/saltstack/salt/pull/52191

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/53022

### Tests written?

Yes

### Commits signed with GPG?

Yes

### CODE REVIEW

**Review by dwoz — APPROVED** [review]
Nicely done!

File: tests/unit/modules/test_chocolatey.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+from tests.support.mock import MagicMock, patch
+
+if salt.utils.is_windows():
+    CHOCO_PATH = 'C:\\path\\to\\chocolatey.exe'
+    CHOCO_PATH_PD = os.path.join(
+        os.environ.get('ProgramData'), 'Chocolatey', 'bin', 'chocolatey.exe')
+    CHOCO_PATH_SD = os.path.join(
+        os.environ.get('SystemDrive'), 'Chocolatey', 'bin', 'chocolatey.bat')
+
+    MOCK_FALSE = MagicMock(return_value=False)
```
Comment (s0undt3ch) [inline tests/unit/modules/test_chocolatey.py]: The rest of the globals are OK, but this mock object should be initialized in the test class, `setUpClass` for example.

Comment (s0undt3ch) [reply tests/unit/modules/test_chocolatey.py]: Ideally, all globals :)

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/unit/modules/test_chocolatey.py (hunk 1 above)
Comment (twangboy) [reply tests/unit/modules/test_chocolatey.py]: Done

**Review by s0undt3ch — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Just a little lint and we're done.

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**